src/Components/Nav/Components/Category.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './Category.scss';
+
+class Category extends Component {
+  constructor() {
+    super();
+    this.state = {
+      categoryList: [],
+      isSubOpen: false,
+      categoryIdx: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/category.json', {
+      method: 'GET',
+    })
```

Comment:
```suggestion
  componentDidMount() {
    fetch('/data/category.json')
```
- get method는 default기 때문에 생략할 수 있습니다
- public 폴더에서 가져오는 데이터의 경우 url과 port를 생략해줄 수 있습니다

Reply:
넵 감사합니다.